### ISSUE TITLE
Divergence with PT over capture groups in PrintDraftChanges.txt in: clauses

### ISSUE THREAD

**Issue by davidg-sil** [sillsdev/ptx2pdf#227]
With there being so little help in PT on the correct format for limited-range changes, I ignorantly I had the following lines in my PrintDraftChanges.txt
```
# Quotes in footnotes are not dialogue
in "\\f( .*)\\f\*": '<<' > '„'
in "\\f( .*)\\f\*": '>>' > '”'
# Dash-style dialogue markup
'<<'	> '\\p  — ' #  Dialoglue start a dashed paragraph
'>> *(\\f .*\\f\*)' > '\1\\p' # Dialogue end footnote  goes before the break
'>> *(\\x .*\\x\*)' > '\1\\p' # Dialogue end xref goes before the break 
'>>'	> '\\p' 	#Dialoglue end with a paragraph.
```

In Paratext, the capturing brackets in the top 2 rules  (with ```in:```), which I guessed might indicate the context to apply changes to) were either taken as how I guessed, or were ignored. 

In ptx2pdf, 
```
\f + \fr 1:20 \fq Naomi\fq* \ft hakeardeol and-e cib le evreenghi <<plăkuto>>, hai \fq Mara\fq* hakeardeol <<kerkimos>>.\f*
```
became:
```
\f + \fr 1:20 \fq Naomi\fq* \ft hakeardeol and-e cib le evreenghi „plăkuto”, hai \fq Mara\fq* hakeardeol „kerkimos”.\f*
 + \fr 1:20 \fq Naomi\fq* \ft hakeardeol and-e cib le evreenghi „plăkuto\p, hai \fq Mara\fq* hakeardeol „kerkimos\p.
 + \fr 1:20 \fq Naomi\fq* \ft hakeardeol and-e cib le evreenghi \p — plăkuto”, hai \fq Mara\fq* hakeardeol \p — kerkimos”
```
Indicating that the captured text from the `in:` clause ended up being repeated after the substituted text. I can't think of any time this behaviour might be considered a feature, so I'm guessing it counts as a bug.

**Comment by mhosken** [comment]
My reading of Paratext is that the capture groups in the in clause are ignored. We say they are illegal and quietly do bad things with them. Either way. They certainly aren't doing what you may think they are doing. So I recommend removing them.

Having said that, I notice that PT supports two `in` clauses, and we don't, and also we should do a better job of ignoring any groupings someone may put into their change.

**Comment by mhosken** [comment]
Fixed? in 1.2.1
